sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/parser/ExpressionParserSuite.scala

```diff
@@ -329,6 +330,27 @@ class ExpressionParserSuite extends AnalysisTest {
       parameters = Map("error" -> "'x'", "hint" -> ": extra input 'x'"))
   }
 
+  test("function expressions with named arguments") {
+    assertEqual("encode(value => 'abc', charset => 'utf-8')",
+                $"encode".function(NamedArgumentExpression("value", Literal("abc")),
+                  NamedArgumentExpression("charset", Literal("utf-8"))))
```

Comment:
Please, fix indentation here and below according to https://github.com/databricks/scala-style-guide
```suggestion
      $"encode".function(NamedArgumentExpression("value", Literal("abc")),
      NamedArgumentExpression("charset", Literal("utf-8"))))
```